content/docs/reference/glossary.md

```diff
@@ -136,6 +136,10 @@ A Neon component which functions as a multitenant service that accepts and handl
 
 A Neon feature that allows users to connect to a Neon project with a single `psql` command. See [Query with psql](/docs/get-started-with-neon/query-with-psql-editor/).
 
+### Region
+
+The geographic location where Neon project resource are located. Neon supports creating projects in several Amazon Web Services (AWS) regions. For information about regions supported by Neon, see [Regions](../../conceptual-guides/regions).
```

Comment:
Can we add that the Neon project should be close to the user for better latency?

Reply:
We could, but I do mention that in the Regions topic: https://github.com/neondatabase/website/pull/250